app/models/image.rb

```diff
@@ -30,4 +32,12 @@ def set_image_data_from_content
   def decoded_image
     @decoded_image ||= Base64.decode64(content)
   end
+
+  private
+
+  def image_size
+    if decoded_image.bytes.count > MAX_IMAGE_SIZE
+      raise Catalog::InvalidParameter, "Image too large, must be #{MAX_IMAGE_SIZE.to_s(:human_size)} or lower"
```

Comment:
Error message suggestion: `"Image size exceeds max limit of #{MAX_IMAGE_SIZE.to_s(:human_size)}"`

Reply:
Sounds much better. Updated!